style/README.md

```diff
@@ -39,9 +39,6 @@ Formatting
 * Don't include spaces after `(`, `[` or before `]`, `)`.
 * Don't misspell.
 * Don't vertically align tokens on consecutive lines.
-* If you break up an argument list, keep the arguments on their own lines and
-  closing parenthesis on its own line. [Example 1][multi-line arguments]
```

Comment:
Sadly, how the samples are organized. A few links might have been broken since they are done by line-number. That's a shame